src/plugins/explore/public/application/utils/state_management/slices/query_editor/query_editor_slice.ts

```diff
@@ -46,6 +47,7 @@ const initialState: QueryEditorSliceState = {
   dateRange: undefined,
   isQueryEditorDirty: false,
   hasUserInitiatedQuery: false,
+  currentEditorQuery: '',
```

Comment:
we intentionally took out the editor query state out of redux due to performance issues. if you originally did this to get the current editor text, i suggest `useEditorText()`: https://github.com/angle943/OpenSearch-Dashboards/blob/8eff1b6ec32de66ce1c29cb0b79c3f7bccb4e200/src/plugins/explore/public/application/hooks/editor_hooks/use_editor_text/use_editor_text.ts

Reply:
Reverted this change, using now `useEditorText` to get the current query  [c984e86](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/10849/commits/c984e864a83e320b3766b2dad83a52b315858e66)